internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -82,6 +82,25 @@ Resources:
           ImageRepositoryType: !Ref ImageRepositoryType
           ImageConfiguration:
             Port: !Ref ContainerPort
+            {{- if hasSecrets .}}
```

Comment:
Good call out. I have intentionally written this field definition in the cf.yml because for _RDWS_ service the CFN template should have **RuntimeEnvironmentSecrets** as the field heading instead of **Secrets**(which is the case for _ECS_ services)